src/System.Runtime.Extensions/tests/System/Environment.GetEnvironmentVariable.cs

```diff
@@ -170,6 +170,7 @@ public void GetEnumerator_IDictionaryEnumerator_YieldsDictionaryEntries()
         [Theory]
         [InlineData(null)]
         [MemberData(nameof(EnvironmentTests.EnvironmentVariableTargets), MemberType = typeof(EnvironmentTests))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/23003")]
```

Comment:
Please disable it using ActiveIssueAttribute: `[ActiveIssue("https://github.com/dotnet/corefx/issues/23003", TargetFrameworkMonikers.NetFramework)]`

`SkipOnTargetFrameworkAttribute` is intended to skip the test permanently because there is a difference in behavior. `ActiveIssueAttribute` is intended for temporarily disabling tests that have a related issue and has to be fixed.